packages/trader/src/App/Containers/Modals/success-dialog.jsx

```diff
@@ -94,10 +94,14 @@ SuccessDialog.propTypes = {
     icon: PropTypes.object,
     icon_size: PropTypes.string,
     icon_type: PropTypes.string,
+    is_medium_button: PropTypes.bool,
```

Comment:
I added these props here to avoid another TS error saying that 'SuccessDialog is missing the following (required) properties: text_cancel, is_medium_button'.